src/components/RadialMenu.svelte

```diff
@@ -140,6 +149,10 @@
 		clickCoords = null;
 		mouseCoords = null;
 		document.body.style.cursor = 'initial';
+		setBodyUserSelect('initial');
+	}}
+	on:touchend={() => {
+		setBodyUserSelect('initial');
```

Comment:
This will override the existing touchend method

Reply:
Fixed in HEAD